src/_actions/paymentActions.js

```diff
@@ -10,16 +10,21 @@ import {
   MAKE_PAYMENT_REQUEST,
   MAKE_PAYMENT_SUCCESS,
 } from "../_constants/paymentsConstant";
+const Cookie = require("js-cookie");
 
 const makePayment = (phone, amount) => async (dispatch) => {
   dispatch({ type: MAKE_PAYMENT_REQUEST, payload: { phone, amount } });
   try {
     await axios.post("/mpes/stk", { phone, amount })
     .then(response=>{
       console.log(response)
+      Cookie.set('paymentDetails', JSON.stringify(response.data),{
+        expires:1/28800
+
+      })
     dispatch({ type: MAKE_PAYMENT_SUCCESS, payload: response });
 
-    })
+    }).catch(err=>console.log(err))
```

Comment:
Codacy has a fix for the issue: [Missing space before =>.](https://app.codacy.com/gh/bramuel/kitenge-recap/pullRequest?prid=7272348)
```suggestion
    }).catch(err =>console.log(err))
```